ClosedXML_Tests/Excel/Misc/XLWorkbookTests.cs

```diff
@@ -418,5 +418,28 @@ public void WbProtect5()
                 Assert.IsFalse(wb.IsPasswordProtected);
             }
         }
+
+        [Test]
+        public void FileSharingProperties()
+        {
+            using (var ms = new MemoryStream())
+            {
+                using (var wb = new XLWorkbook())
+                {
+                    wb.AddWorksheet("Sheet1").Cell("A1").Value = "Hello world!";
+                    wb.FileSharing.ReadOnlyRecommended = true;
+                    wb.FileSharing.UserName = Environment.UserName;
```

Comment:
Better use some dummy value then rely on the environment.

Reply:
This was actually deliberate, because I wanted to see whether it will work. And it does. So we might as well leave it? This will probably be the typical use case anyway.

Reply:
OK, let's leave it